controllers/node_controller.go

```diff
@@ -123,11 +124,64 @@ func (r *NodeReconciler) doFinalize(ctx context.Context, log logr.Logger, node *
 			return ctrl.Result{}, err
 		}
 		log.Info("deleted PVC", "name", pvc.Name, "namespace", pvc.Namespace)
+
+	}
+
+	lvList := new(topolvmv1.LogicalVolumeList)
+	err = r.List(ctx, lvList, client.MatchingFields{KeyLogicalVolumeNode: node.Name})
+	if err != nil {
+		log.Error(err, "failed to get LogicalVolumes")
+		return ctrl.Result{}, err
+	}
+
+	for _, lv := range lvList.Items {
+		err = r.cleanupLogicalVolume(ctx, log, &lv)
+		if err != nil {
+			log.Error(err, "unable to cleanup LogicalVolume")
```

Comment:
This error message is redundant. Because error messages are logged in the function.

Reply:
fixed it.